generators/app/templates/src/app/components/Routes/components/AuthenticatedRoute.tsx

```diff
@@ -74,11 +80,4 @@ AuthenticatedRoute.defaultProps = {
   currentUser: false
 };
 
-AuthenticatedRoute.propTypes = {
-  ...Route.propTypes, // eslint-disable-line react/forbid-foreign-prop-types
-  currentUser: PropTypes.bool,
-  isPrivateRoute: PropTypes.bool,
-  isPublicRoute: PropTypes.bool
-};
-
-export default withRouter(connect()(AuthenticatedRoute));
+export default connect()(AuthenticatedRoute);
```

Comment:
I think we can remove the `connect` if it's not being used, I know you didn't do it but since you'll be making changes...